web/src/styles/scss/password-meter.scss

```diff
@@ -2,12 +2,12 @@
 
 // This code comes from https://github.com/gladchinda/password-strength-react-demo/blob/master/src/App.css#L45
 
-.password-with-meter {
+.password-input {
```

Comment:
Since the meter's optional now, it seemed odd to have the classname reflect the meter disirregardless.